src/js/jsx/sections/style/Fill.jsx

```diff
@@ -168,12 +167,10 @@ define(function (require, exports, module) {
          * @param {boolean} isChecked
          */
         _toggleFillEnabled: function (event, isChecked) {
-            var color = this.props.fill && collection.uniformValue(this.props.fill.colors) || Color.DEFAULT;
-
             this.getFlux().actions.shapes.setFillEnabled(
                 this.props.document,
                 this.props.layers,
-                color,
+                null,
```

Comment:
I think you can just remove this argument entirely from this method, and just change the shape action accordingly.  Same for `setStrokeEnabled`